Cabal-tests/tests/UnitTests/Distribution/Version.hs

```diff
@@ -44,7 +43,6 @@ versionTests =
     , tp "normaliseVersionRange involutive"                    prop_normalise_inv
     , tp "normaliseVersionRange equivalent"                    prop_normalise_equiv
     , tp "normaliseVersionRange caretequiv"                    prop_normalise_caret_equiv
-    , tp "normaliseVersionRange model"                         prop_normalise_model
```

Comment:
Why is this test being removed?

Reply:
Ah, this was removed because the underlying code was going away as was part of the larger f009c2e originally. I can clean this up in either direction by reinstating the test or by adding to the removals. Any thoughts on the direction?

Reply:
Maybe, to get this PR merged ASAP, before it bit-rots, let's re-add the test and leave the extra removal for future consideration?

Reply:
I concur